apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -61,7 +67,8 @@ class TeacherFeedback extends Component {
       studentId: studentId,
       latestFeedback: [],
       submitting: false,
-      errorState: false
+      errorState: ErrorType.NoError,
+      token: ""
```

Comment:
My natural inclination is to use the empty string since we'll eventually replace it with a string and I'm coming from a more strongly typed background. Do we have a style preference on this kind of initialization?

Reply:
I don't see how a strong type background changes anything. Strongly typed languages use `null` too. 

In Java for instance, it would generally be proper to initialize a `String` type to `null` if it has no value. In that context, `null` means uninitialized. Empty string is an instance of a String with no contents, not the same as uninitialized / no value. 

Ultimately it comes down to preference though. I don't feel too strongly, but personally I prefer clearly differentiating no value (`null` / `nil` / `undefined`) and empty values (`""`, `[]`, `{}`, `0`)